Essentials/src/com/earth2me/essentials/utils/MaterialUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.earth2me.essentials.utils;
+
+import org.bukkit.Material;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.earth2me.essentials.utils.EnumUtil.getMaterial;
+
+public class MaterialUtil {
+
+    private static final Set<Material> BEDS = new HashSet<>();
+
```

Comment:
Add static constants for SIGN_POST and SPAWNER since they're used all over the place.